eliot-service/eliot/app.py

```diff
@@ -143,6 +144,9 @@ def setup(self):
         # Set up Sentry exception logger if we're so configured
         setup_sentry_logging()
 
+        # Set up uncaught error handler
```

Comment:
This probably doesn't need a comment but it seemed aesthetically nicer given the lines around it all have comments.